commands/serve.js

```diff
@@ -1,88 +1,49 @@
 'use strict';
 
+var _ = require('lodash');
 var chalk = require('chalk');
-var RSVP = require('rsvp');
-var superstatic = require('superstatic').server;
 
 var Command = require('../lib/command');
-var detectProjectRoot = require('../lib/detectProjectRoot');
-var FirebaseError = require('../lib/error');
-var hostingImplicitInit = require('../lib/hostingImplicitInit');
-var hostingInitMiddleware = require('../lib/hostingInitMiddleware');
 var logger = require('../lib/logger');
 var utils = require('../lib/utils');
 var requireAccess = require('../lib/requireAccess');
 var requireConfig = require('../lib/requireConfig');
 var checkDupHostingKeys = require('../lib/checkDupHostingKeys');
+var serve = require('../lib/serve/index');
+var VALID_TARGETS = ['functions', 'hosting'];
```

Comment:
Nit: Let's move this below `scopes` with a line break between. Typically I like to do:

1. Module dependencies
2. Local dependencies
3. Constant declaration